source/jquery.flot.js

```diff
@@ -2492,10 +2494,33 @@ Licensed under the MIT license.
             }
         }
 
-        // returns the data item the mouse is over/ the cursor is closest to, or null if none is found
+        function findNearbyItems(mouseX, mouseY, seriesFilter, radius, computeDistance) {
+            var items = findItems(mouseX, mouseY, seriesFilter, radius, computeDistance);
+            executeHooks(hooks.findNearbyItems, [mouseX, mouseY, seriesFilter, radius, computeDistance, items]);
+            return items;
+        }
+
         function findNearbyItem(mouseX, mouseY, seriesFilter, radius, computeDistance) {
-            var i, j,
-                item = null,
+            var items = findNearbyItems(mouseX, mouseY, seriesFilter, radius, computeDistance);
+            var closest = items[0];
+ 
+            for (var i = 1; i < items.length; ++i) {
+               if (closest.distance != undefined && items[i].distance < closest.distance) {
```

Comment:
Why are we not checking for an undefined distance on `items[i]`? Also, it seems like we're treating an item with an undefined distance as the "closest" here, but we're treating it as the farthest in the hover plugin.

Reply:
Fixed.